acceptance: TestDockerCLI failed

[(acceptance).TestDockerCLI failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1712793&tab=buildLog) on [master@a3cb002a3a43c70afdce15c1a7670bb8f59c75ac](https://github.com/cockroachdb/cockroach/commits/a3cb002a3a43c70afdce15c1a7670bb8f59c75ac):

```
=== RUN   TestDockerCLI
--- FAIL: TestDockerCLI (336.72s)
    test_log_scope.go:77: test logs captured to: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/logTestDockerCLI304352059
    test_log_scope.go:58: use -show-logs to present logs inline
=== RUN   TestDockerCLI/test_multiple_nodes.tcl
    --- FAIL: TestDockerCLI/test_multiple_nodes.tcl (34.19s)

```

<details><summary>More</summary><p>
Parameters:

- GOFLAGS=-json

```
make stressrace TESTS=TestDockerCLI PKG=./pkg/acceptance TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```


[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestDockerCLI.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(acceptance).TestDockerCLI failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1713316&tab=buildLog) on [master@9fb0e302c5a3775c4f3ec119aeed40a4bd1a65bc](https://github.com/cockroachdb/cockroach/commits/9fb0e302c5a3775c4f3ec119aeed40a4bd1a65bc):

```
=== RUN   TestDockerCLI
--- FAIL: TestDockerCLI (335.68s)
    test_log_scope.go:77: test logs captured to: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/logTestDockerCLI791514583
    test_log_scope.go:58: use -show-logs to present logs inline
=== RUN   TestDockerCLI/test_multiple_nodes.tcl
    --- FAIL: TestDockerCLI/test_multiple_nodes.tcl (33.74s)

```

<details><summary>More</summary><p>
Parameters:

- GOFLAGS=-json

```
make stressrace TESTS=TestDockerCLI PKG=./pkg/acceptance TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```


[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestDockerCLI.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(acceptance).TestDockerCLI failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1713759&tab=buildLog) on [master@85df0c217e4ca067962399c0cf3f087698b1354e](https://github.com/cockroachdb/cockroach/commits/85df0c217e4ca067962399c0cf3f087698b1354e):

```
=== RUN   TestDockerCLI
--- FAIL: TestDockerCLI (330.53s)
    test_log_scope.go:77: test logs captured to: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/logTestDockerCLI112337256
    test_log_scope.go:58: use -show-logs to present logs inline
=== RUN   TestDockerCLI/test_multiple_nodes.tcl
    --- FAIL: TestDockerCLI/test_multiple_nodes.tcl (35.09s)

```

<details><summary>More</summary><p>
Parameters:

- GOFLAGS=-json

```
make stressrace TESTS=TestDockerCLI PKG=./pkg/acceptance TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```


[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestDockerCLI.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(acceptance).TestDockerCLI failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1713880&tab=buildLog) on [master@8c5838d7344ec3516325e1e4cd5e8e34c04e91c9](https://github.com/cockroachdb/cockroach/commits/8c5838d7344ec3516325e1e4cd5e8e34c04e91c9):

```
=== RUN   TestDockerCLI
--- FAIL: TestDockerCLI (338.20s)
    test_log_scope.go:77: test logs captured to: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/logTestDockerCLI293375765
    test_log_scope.go:58: use -show-logs to present logs inline
=== RUN   TestDockerCLI/test_multiple_nodes.tcl
    --- FAIL: TestDockerCLI/test_multiple_nodes.tcl (33.87s)

```

<details><summary>More</summary><p>
Parameters:

- GOFLAGS=-json

```
make stressrace TESTS=TestDockerCLI PKG=./pkg/acceptance TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```


[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestDockerCLI.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

--- a/pkg/cli/interactive_tests/test_multiple_nodes.tcl
+++ b/pkg/cli/interactive_tests/test_multiple_nodes.tcl
@@ -22,10 +22,10 @@ end_test
 
 
 start_test "Check that a double decommission prints out a warning"
-spawn $argv node decommission 2
+spawn $argv node decommission 2 --wait none
 eexpect eof
 
-spawn $argv node decommission 2
+spawn $argv node decommission 2 --wait none
 eexpect "warning: node 2 is already decommissioning or decommissioned"
 eexpect eof
 end_test

